lib/admin_page/user/mobile_and_web_layout_design/web_dsign.dart

```diff
@@ -0,0 +1,296 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:web/model/user_data.dart';
+
+import '../../../values/app_assets.dart';
+import '../widget/widget_user_detail.dart';
+
+class DesktopPage extends StatefulWidget {
+ DesktopPage({Key? key, this.user}) : super(key: key);
+  UserData? user;
+  @override
+  State<DesktopPage> createState() => _DesktopPageState();
+}
+
+class _DesktopPageState extends State<DesktopPage> {
+  @override
+  Widget build(BuildContext context) {
+    double height = MediaQuery.of(context).size.height;
+    double width = MediaQuery.of(context).size.width;
+    return Scaffold(
+      backgroundColor: Colors.cyanAccent.withOpacity(0.4),
+      body: Stack(
+        children: [
+          Text(
+            "+++$width++",
+            style: const TextStyle(
+                color: Colors.black, fontWeight: FontWeight.bold, fontSize: 16),
+          ),
+          Container(
+            decoration: const BoxDecoration(
+              image: DecorationImage(
+                  image: AssetImage(AppAssets.backGroundImage),
+                  fit: BoxFit.cover),
+            ),
+            child: Padding(
+              padding: EdgeInsets.only(
+                  top: height * 0.14,
+                  bottom: height * 0.14,
+                  left: width * 0.1,
+                  right: width * 0.1),
+              child: Container(
+                width: width,
+                height: height,
+                decoration: BoxDecoration(color: Colors.white, boxShadow: [
+                  BoxShadow(
+                    offset: const Offset(0.5, 0.5),
+                    color: Colors.white.withOpacity(0.8),
+                  ),
+                ]),
+                child: Stack(
+                  children: [
+                    Column(
+                      children: [
+                        Text(
+                          "+++width=$width++ height=$height",
```

Comment:
wrap in debug mode conditioning, It is hard for me to go through all the code and comment for the same issue everywhere, similarly, it may be skipped while submitting the build to production, which will leave a bad effect on the client in terms of UI. So in the next commit, there should be no unnecessary widget just for testing without debug mode wrapper.